### PR TITLE
Cross platform compatibility + other fixes

### PR DESCRIPTION
**Pull request by nhthn**
Thanks for this project, I'm looking forward to using this in my workflow. Some fixes:

- Add instructions for development setup
- Detect sclang correctly on Linux and Windows
- Change "off" emoji to ⭕ for colorblind-friendliness
- Add scvsc.useScideKeybindings option to emulate some SCIDE shortcuts
- Clicking status bar icon boots or quits sclang

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,17 @@ I've only tested this on MacOS.
...
+5. Alternatively, if you want to emulate the SCIDE keybindings, go to File -> Preferences -> Settings -> Extensions -> scvsc and set "Use Scide Keybindings" to `true`.
+
+
+### Installing for development
+
+1. Clone repository.
+2. Run `npm install` in repo root at the command line.
+3. Open `extension.js` in VSCode and press `F5`. Select the option to run the "Extension Development Host."
+4. Open a `.scd` file. You may be prompted to install the HyperScope vscode extension, allow it. You will see an "sclang" indicator in the lower left.
+5. Press `Ctrl+Shift+P` to open the command palette. Search for "SuperCollider" to find options such as "start sclang."
```
Comment (alexander-daniel) [inline README.md:35]: maybe just "open the command palette" since that keybinding might be user defined

File: extension.js [hunk 2]
```diff
@@ -1,10 +1,33 @@
...
 const Lang = require('supercolliderjs').lang.default;
 const { flashHighlight } = require('./util');
 
+const SCLANG_STATUS_BAR = 'sclang';
+const SCLANG_STATUS_BAR_OFF = `${SCLANG_STATUS_BAR} ⭕`;
+const SCLANG_STATUS_BAR_ON = `${SCLANG_STATUS_BAR} 🟢`;
+
 let lang = null;
 
+function getDefaultSclangExecutable() {
```
Comment (alexander-daniel) [inline extension.js:14]: 👍  fantastic, can we pop this into `util`?

File: extension.js [hunk 3]
```diff
@@ -29,68 +54,81 @@ async function activate(context) {
...
-      }
+  async function toggleSclang() {
+    if (lang === null) {
+      startSclang();
+    } else {
+      stopSclang();
     }
+  }
+
+  const startSCLangCommand = vscode.commands.registerCommand(
```
Comment (alexander-daniel) [inline extension.js:115]: ah nice, much cleaner

**Review by alexander-daniel — APPROVED** [review]
Amazing! Thanks for the great changes! I left a few non-blocking comment, but these changes are really really appreciated!

🚚 !

File: README.md (hunk 1 above)
Comment (nhthn) [reply README.md]: I've removed this line entirely, since these are dev instructions I assume people know how to open the command palette

**Comment by nhthn** [comment]
Thanks for the prompt response, I've addressed the comments.

**Review by alexander-daniel — APPROVED** [review]
@nhthn Thank you for considering the changes -- no problem for the version, go ahead and merge and I'll bump it before I publish. I'll ping you on the release ! Thanks again for the improvements.

**Comment by alexander-daniel** [comment]
@nhthn https://github.com/alexander-daniel/scvsc/releases/tag/0.1.0

Let me know if you have any issues!

**Pull request merged**